### PR TITLE
Play NERVE wad with original Doom 2 non BFG Edition wad?

### PR DESCRIPTION
**Pull request by quakeguy123**
Hi Fabian,

I wonder if it's possible in Crispy doom to play NERVE wad with any non BFG Edition doom 2 wad?
I have tried the following: replaced the doom BFG Edition wad with original doom 2 wad, and used the NERVE wad with that wad. Needless to say it did not work.

Let me rephrase my question: Is it possible to tweak the engine to recognize the original doom2.wad as being the BFG Edition wad so that the original doom2.wad can be played uncensored with the NERVE.wad?

### CODE REVIEW

**Comment by fragglet** [comment]
Wrong repository I think.

**Comment by fabiangreffrath** [comment]
Hi quakeguy,

this time you requested Chocolate Doom to merge the entire Crispy Doom -- which is of course non-sense. Please double-check next time that you are filing an "issue" and not a "pull request".

> I wonder if it's possible in Crispy doom to play NERVE wad with any non BFG Edition doom 2 wad?

You mean something as simple as `crispy-doom -iwad doom2.wad -file nerve.wad`?

Yes, of course this is possible and will result in the same game as if you selected NRFTL from the Episode menu that is shown with the BFG Edition DOOM2.WAD. It will not display the Episode menu, though, because the graphics required to render that menu are only available in the BFG Edition DOOM2.WAD file.

If you have any further questions or if the command line I posted above does not work for you, please file an issue (not a pull request) on the Crispy Doom repository.

Thanks, 

Fabian
